lanl/docker-compose/ochami-krakend-ce.yml

```diff
@@ -1,20 +1,43 @@
 version: '3.7'
 
 services:
-  
+  step-ca:
+    hostname: SI.ca
+    container_name: SI.ca
+    image: ghcr.io/openchami/local-ca:acme
+    ports: 
+      - "443:443"
+    healthcheck: 
+      test: ["CMD-SHELL", "step", "ca", "health", "--ca-url", "https://SI.ca", "--root", "$(step path)/certs/root_ca.crt"]
+      interval: 30s
+      timeout: 10s
+      retries: 5
+      start_period: 10s
+    networks:
+      - internal
+    volumes:
+      - ./configs/step-ca:/mnt
   krakend-ce:
-    container_name: krakend-ce
-    image: devopsfaith/krakend
+    hostname: SI.krakend
+    container_name: SI.krakend
+    image: ghcr.io/openchami/krakend:acme
     environment:
       - KRAKEND_CONFIG=/etc/krakend/krakend.json
       - KRAKEND_PORT=8080
-    hostname: krakend-ce
+    healthcheck:
+      test: ["CMD-SHELL", "CURL_CA_BUNDLE=/root/.acme.sh/SI.krakend_ecc/SI.krakend.cer", "curl", "-f", "https:/SI.krakend:8080/__health"]
```

Comment:
Does this indicate that .acme.sh must be present in the /root directory of the VM?  The host machine?  The container?  I think a comment here to clarify which /root and why would be very helpful.